compiler/aarch64/codegen/ControlFlowEvaluator.cpp

```diff
@@ -776,3 +776,27 @@ static bool virtualGuardHelper(TR::Node *node, TR::CodeGenerator *cg)
    return false;
 #endif
    }
+
+TR::Register *OMR::ARM64::TreeEvaluator::igotoEvaluator(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT(node->getNumChildren()>=1, "at least one child expected for igoto");
```

Comment:
If you're going to add an assert, make it a `TR_ASSERT_FATAL` and have it check `node->getNumChildren() <= 2`, which is what you're expecting.

I'm not sure this is necessary though.  Most other evaluators don't verify their child count.  This seems like something the tree verifier should handle.

Reply:
I added this assert because it existed in p/x/z version of igotoEvaluator, but I will remove it to be consistent with other evaluators as you suggested.